client/uploadScrambles.js

```diff
@@ -105,12 +105,11 @@ function extractJsonFromZip(filename, zipId, pw, cb) {
         } else {
           cb("Wrong password");
         }
-        return;
-      } else {
-        throw error;
       }
+      console.error("Meteor.call() error: " + err);
```

Comment:
It looks like the behavior here has changed. Won't this run the console.error(...) even when they just typed in an incorrect password?